net.odamex.Odamex.yml

```diff
@@ -0,0 +1,74 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/flatpak/flatpak-builder/main/data/flatpak-manifest.schema.json
+---
+app-id: net.odamex.Odamex
+runtime: org.freedesktop.Platform
+runtime-version: "24.08"
+sdk: org.freedesktop.Sdk
+command: select-exe
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --device=all # change this to device=input + device=dri when available more widely
+  - --socket=pulseaudio
+  - --share=network
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --filesystem=host
```

Comment:
also why does this require `host` filesystem access.

Reply:
I knew there'd be something I forgot to do before submitting. It shouldn't. I'll investigate what is actually required.

Reply:
Quick question: if I do `--filesystem=/usr/local/share/games/doom:ro` will that be mounted as `/usr/local/share/games/doom` or `/run/host/usr/local/share/games/doom`? And is that the correct place to put `:ro` for read-only?

Reply:
No you can't pick individually.